packages/analytics-js-integrations/src/integrations/Amplitude/browser.js

```diff
@@ -59,9 +62,15 @@ class Amplitude {
     window.amplitude.init(this.apiKey, null, initOptions);
   }
 
-  identify(rudderElement) {
-    logger.debug('in Amplitude identify');
+  isLoaded() {
+    return !!(window.amplitude && window.amplitude.getDeviceId());
```

Comment:
using Boolean() is preffered to !! for brevity

Reply:
updated